File: app/src/main/java/com/close/hook/ads/util/OnClearClickListener.java
package com.close.hook.ads.util;

import com.close.hook.ads.data.model.FilterBean;

public interface OnClearClickListener {

    void onClearAll();

    void search(String keyWord);

    void updateSortList(FilterBean filterBean, String keyWord, Boolean isReverse);

}

File: app/src/main/java/com/close/hook/ads/util/OnClearClickListener.kt
package com.close.hook.ads.util

interface OnClearClickListener {
    fun onClearAll() {}

    fun search(keyWord: String) {}

    fun updateSortList(filterOrder: Int, keyWord: String, isReverse: Boolean, showConfigured: Boolean, showUpdated: Boolean, showDisabled: Boolean) {}
}
